internal/cel/transformer.go

```diff
@@ -0,0 +1,45 @@
+package transformer
+
+import (
+	"errors"
+
+	//"github.com/google/cel-go/checker/decls"
+	"github.com/google/cel-go/common"
+	"github.com/google/cel-go/common/ast"
+	//"github.com/google/cel-go/common/types/ref"
+	//"github.com/google/cel-go/ext"
+	"github.com/google/cel-go/parser"
+)
+
+func parseExpression(expression string) (*ast.AST, error) {
+	prsr, err := parser.NewParser()
+	if err != nil {
+		return nil, err
+	}
+
+	p, iss := prsr.Parse(common.NewTextSource(expression))
+	if len(iss.GetErrors()) > 0 {
+		return nil, errors.New("got errors parsing")
+	}
+
+	return p, nil
+}
+
+func SafeToSimplyPrefix(expression string) bool {
```

Comment:
This probably makes it a whole lot safer already... while I try to get an AST transformation going.